docs/source/data/data_catalog.md

```diff
@@ -32,21 +32,21 @@ shuttles:
 ```
 ### Dataset `type`
 
-Kedro offers a range of datasets, including CSV, Excel, Parquet, Feather, HDF5, JSON, Pickle, SQL Tables, SQL Queries, Spark DataFrames and more. They are supported with the APIs of pandas, spark, networkx, matplotlib, yaml and more.
+Kedro supports a range of connectors, for CSV files, Excel spreadsheets, Parquet files, Feather files, HDF5 files, JSON documents, pickled objects, SQL tables, SQL queries, and more. They are supported using libraries such as pandas, PySpark, NetworkX, and Matplotlib.
```

Comment:
```suggestion
Kedro supports a range of connectors, for CSV files, Excel spreadsheets, Parquet files, Feather files, HDF5 files, JSON documents, pickled objects, SQL tables, SQL queries and more. They are supported using libraries such as pandas, PySpark, NetworkX and Matplotlib.
```
Remove Oxford commas?

Reply:
Let's leave them! I am (relatively) happy to use them for clarity and as Vale spots them missing throughout the docs in weeks to come, I'll add them in rather than take them out.